domain/service_metadata.go

```diff
@@ -23,7 +22,7 @@ func (sm ServiceMetadata) MarshalJSON() ([]byte, error) {
 
 	b, err := json.Marshal(Alias(sm))
 	if err != nil {
-		return []byte{}, errors.Wrap(err, "unmarshallable content in AdditionalMetadata")
+		return nil, fmt.Errorf("unmarshallable content in AdditionalMetadata: %w", err)
 	}
 
 	var m map[string]interface{}
```

Comment:
The error returned by json.Unmarshal operation in line 29 is not being checked. It's a good practice in Go to always check for errors. I know it is out of the scope of the PR, but you may want to add the logic

Reply:
Thanks @zucchinidev. I'll handle this in https://github.com/pivotal-cf/brokerapi/pull/303